apps/web/lib/getTemporaryOrgRedirect.ts

```diff
@@ -0,0 +1,44 @@
+import logger from "@calcom/lib/logger";
+import { safeStringify } from "@calcom/lib/safeStringify";
+import type { RedirectType } from "@calcom/prisma/client";
+
+const log = logger.getChildLogger({ prefix: ["lib", "getTemporaryOrgRedirect"] });
+export const getTemporaryOrgRedirect = async ({
+  slug,
+  redirectType,
+  eventTypeSlug,
+}: {
+  slug: string;
+  redirectType: RedirectType;
+  eventTypeSlug: string | null;
+}) => {
+  const prisma = (await import("@calcom/prisma")).default;
+  log.debug(
+    `Looking for redirect for`,
+    safeStringify({
+      slug,
+      redirectType,
+      eventTypeSlug,
+    })
+  );
+  const redirect = await prisma.tempOrgRedirect.findUnique({
```

Comment:
It's a temporary table and would be really lightweight. So, It should be quick to query but there would still be DB latency that would be added in querying a booking page.

I feel it's okay to have it for now, till we actually see a significant issue with it(as we will remove it anyway soon when mono-user Org schema is ready)

If we see an issue, our option then is to query this in parallel, with other queries on Booking Page which is possible